try {
   console.log(testando)
    }
    catch (error) {
      console.error("Ops, isso n√£o deveria acontecer, contate um desenvolvedor para que ele possa reparar esse erro!");
      console.log("")
      console.log(error)
    }
   
